### ISSUE TITLE
Compatibility tool configuration failed with Proton 5.13-5 on Manjaro (archlinux)

### ISSUE THREAD

**Issue by dnezamaev** [ValveSoftware/steam-runtime#364]
# Compatibility Report
- Name of the game with compatibility issues: Trine 4 The Nightmare Prince
- Steam AppID of the game: 690640

## System Information
- GPU: GeForce GTX 1050 Ti Mobile + Intel Corporation UHD Graphics 630
- Driver/LLVM version: video-hybrid-intel-nvidia-prime from official Manjaro rep (nvidia 460 i think)
- Kernel version: 5.4.89-1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/dnezamaev/fbe795559a8badaf1f78b468b50ba34c
- Proton version: 5.13-5

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [x ] that I have checked whether there are updates for my system available.

No file generated, proton was not started.

## Symptoms 

When I use latest Proton 5.13-5 for my Steam client to launch Trine 4 it says "An error occurred while updating ... (compatibility tool configuration failed)". I followed suggested link to troubleshooting and tried to restart computer, reinstall steam, install latest NVidia drivers, update system, use stable LTS or latest kernel. Also checked that I have no antivirus, firewall or other interfering software.

When I use Proton 5.0-10 or 4.11-13 or 4.2-9 Trine 4 updates successfully and even tries to start, but ends in black screen with in-game cursors.

## Reproduction

Install Manjaro KDE, Steam, try to start Trine 4.

**Comment by kisak-valve** [comment]
Thanks, that tells us that it's more likely that there's an issue with Pressure Vessel than the Steam Client. Pressure Vessel is the tool that sets up the container environment for Proton to use. Please give https://github.com/ValveSoftware/steam-runtime/blob/master/doc/reporting-steamlinuxruntime-bugs.md#essential-information a read and share a Steam Linux Runtime log if you're able to generate one.

**Comment by dnezamaev** [comment]
Launched "STEAM_LINUX_RUNTIME_LOG=1 steam" from Konsole. But no *.log files or var directory  appeared in ~/.local/share/Steam/steamapps/common/SteamLinuxRuntime_soldier

Here is console output: https://gist.github.com/dnezamaev/e5ece1c028d084f0473570f923dffaee

Also tried "Older method" with "PRESSURE_VESSEL_VERBOSE=1 steam 2>&1 | tee ~/pressure-vessel.log"

Here is file output: https://gist.github.com/dnezamaev/bf39dc45a66b098a84f7c79f922d9cae

**Comment by kisak-valve** [comment]
Once last quick check before more skilled eyes take a look... if you go to Steam's library view, and right click on `Steam Linux Runtime - Soldier` (or select it and click the gear icon on the right) -> `Properties...` -> `Local Files` -> `Verify integrity of game files...`, does that get Steam to redownload any files?

**Comment by dnezamaev** [comment]
Got "Validating Steam files - 100% complete" in message box header text and "63 files failed to validate and will be reacquired" in body.

On second attempt - "All files successfully validated".

Trine 4 still not starting with same error - compatibility tool configuration failed.

**Comment by 7HEPOW** [comment]
Had the same Issue on Manjaro with games running Proton 5.13 (and newer from GE). Launching the games did basically nothing.

I checked the Runtime Soldier and it was using the client_beta. After opting out of the beta, all games work again. Unfortunately I got no log files to give, but this might be a hint already... :wink:

**Comment by smcv** [comment]
> Got "Validating Steam files - 100% complete" in message box header text and "63 files failed to validate and will be reacquired" in body.
>
> On second attempt - "All files successfully validated".

I think that means it wasn't downloaded correctly the first time, which would be a bug in the Steam client rather than the runtime.

> Trine 4 still not starting with same error - compatibility tool configuration failed

Please could you try again at getting a log?

> I checked the Runtime Soldier and it was using the client_beta. After opting out of the beta, all games work again. Unfortunately I got no log files to give, but this might be a hint already

Sorry, there's nothing we can do with this amount of information. If the `client_beta` is not working, then that's a bug that should be fixed, but we are not going to be able to fix anything without a log.

**Comment by dnezamaev** [comment]
Sorry, can't provide more information now, because I removed Manjaro and installed kUbuntu 20.04.2 LTS. But before it I checked `client_beta` option - it was not set. On kUbuntu I had same problem with last Proton 5.13-5, but after few system restarts it was solved. I don't know how - just installed some other software and Steam games, switched Trine 4 to Proton 5.0-10 and played it a few hours. On next day it worked fine with 5.13-5, have no idea how it happened.

**Comment by kisak-valve** [comment]
Hello @dnezamaev, glad to hear you got it working, but that also means we no longer have a test case to ponder. Closing.

**Comment by enorbet2** [comment]
Just FTR I'm having  the same basic problem on Slackwaree Current... in fact multiple installs of it.  Proton games have worked for more than a thousand hours, then 1 day nothing but "compatibility tool config failed".  I'll try again today on an old 14.2 install just to rule out any distro update issue.  In my case however NO versions of Proton from 4 something up to Experimental work.  One game, Witcher 3 (a great game btw) ran perfectly for hundreds of hours.  Now, and for about 2 months now, only runs in WINE w/ DXVK.

**Comment by smcv** [comment]
>  In my case however NO versions of Proton from 4 something up to Experimental work

That would indicate a problem that is not specific to the Steam Linux Runtime container mechanism, because Proton 4.x has never used the container.

Please report a separate issue with full details.

**Comment by MayeulC** [comment]
I had the same issues yesterday  on one of my installs (I share a drive with part of my steam library between different Arch installs, some use flatpak, etc).

I found the answer here: [https://forum.manjaro.org/t/steam-compatibility-tool-configuration-failed/76193](https://steamcommunity.com/linkfilter/?url=https://forum.manjaro.org/t/steam-compatibility-tool-configuration-failed/76193)

It turns out "Use Steam Play for all other titles" (in Steam Settings -> SteamPlay) **got unchecked** for some reason. I was able to restore that by playing with shader cache and steam play settings.

This happened on the flatpaked steam; I think every steam instance is opted into the beta. Community builds of proton were working fine, but every native instance way displaying that issue.

Weirdly, Steam seems to have deleted some of my downloaded games when I booted it up with SteamPlay disabled.
